src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -85,7 +85,7 @@
     <PackageReference Include="System.ServiceModel.Primitives" Version="4.3.0" />
     <PackageReference Include="System.ServiceModel.Security" Version="4.3.0" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.3.0" />
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.3.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.0.11" />
```

Comment:
With 4.0.11 I can compile locally on my computer. Now I changed to 4.0.1 but failed too.